static/js/github-create-branch.js

```diff
@@ -217,7 +217,7 @@ const createBranchPost = () => {
     })
     .fail((error) => {
       toggleSubmitDisabled(false);
-      showErrorMessage(error.responseJSON);
```

Comment:
So, we did expected a JSON?...

Reply:
It works either way in the frontend, if its just a string we can pull it as `error.responseText`, but if its a JSON, then we pull it as `error.responseJSON` then get the desired property of that object.